api/src/main/java/io/opentelemetry/trace/Tracestate.java

```diff
@@ -235,28 +235,35 @@ public static Entry create(String key, String value) {
 
   // Key is opaque string up to 256 characters printable. It MUST begin with a lowercase letter, and
   // can only contain lowercase letters a-z, digits 0-9, underscores _, dashes -, asterisks *, and
-  // forward slashes /.
+  // forward slashes /.  For multi-tenant vendor scenarios, an at sign (@) can be used to prefix the
```

Comment:
Is the `@` intended to denote a prefix or suffix?  Based on your test below, it seems more of a suffix.

Reply:
it's a prefix for the vendor name (such as `12345@dd`)